app/models/manageiq/providers/vmware/manager_events_mixin.rb

```diff
@@ -0,0 +1,30 @@
+module ManageIQ::Providers::Vmware::ManagerEventsMixin
```

Comment:
Push this down to the `CloudManager` namespace.  There's a direct reference to the `CloudManager` below, which prevents this from being used by any other manager in vmware.

Also, I don't see the infra manager using AMQP for events, so I doubt it will ever be shared anyway.